tests/tensorflow2/test_keras.py

```diff
@@ -395,11 +395,12 @@ def test_keras_fit(out_dir, tf_eager_mode, saveall):
     # can't save gradients in TF 2.x eager mode
     if saveall:  # save losses, metrics, weights, biases
         if tf_eager_mode:
-            assert len(trial.tensor_names()) == (7 if is_tf_2_2() else 8)
+            assert len(trial.tensor_names()) == (12 if is_tf_2_2() else 13)
```

Comment:
is this true in case of both, vanilla TF and custom TF?

Reply:
Yes